src/main/java/edu/kis/vh/nursery/Rhymer.java

```diff
@@ -0,0 +1,15 @@
+package edu.kis.vh.nursery;
+
+public interface Rhymer {
```

Comment:
Incorrect name and operations for implementation hierarchy (good for rhymers hierarchy, i.e., abstraction)